core/src/swarm.rs

```diff
@@ -222,12 +224,13 @@ where
 }
 
 /// Future that must be driven to completion in order for the swarm to work.
-pub struct SwarmFuture<T, H>
+// TODO: rename to `SwarmStream`
```

Comment:
May I suggest `SwarmEvents` or `SwarmEventStream` instead? I think `SwarmStream` is a bit nondescript.

Reply:
I didn't change the name yet in order to not break *too much*, but I guess I will do it anyway.